api_functional_v2_test.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package minio_test
+package minio
 
```

Comment:
Actually, I changed it because I needed access to minPartSize, and that's important to be sure that we are triggering multipart in our tests. I still don't know if there is another way to do that